test/rules/parameter_assignments.dart

```diff
@@ -45,3 +49,34 @@ void otherBad1(int parameter) { // LINT
   parameter += 3;
```

Comment:
Shouldn't the lint be getting reported on line 49 (rather than 48)? Perhaps the location of the "// LINT" comment isn't important, though. (Similarly for some of the other tests.